padinfo/padinfo.py

```diff
@@ -191,10 +191,10 @@ def get_monster_by_id(self, monster_id: int):
         else:
             await ctx.send(self.makeFailureMsg(err))
 
-    @commands.command(name="id")
+    @commands.command(name="id", aliases="iD Id ID".split())
     async def _id(self, ctx, *, query: str):
         """Monster info (main tab)"""
-        await self._do_id(ctx, query)
+        await self._do_id(ctx, query.strip(ctx.prefix+"id"))
```

Comment:
Don't you need a .lower() on query before you do this strip?

Also strip isn't the right thing to use here, I'm always fucking this up too. I probably should audit the usage of strip in the codebase because it's caused bugs periodically.

```
txt = "^id something id"
x = txt.strip("^id")
print(x)
> something
```

Reply:
i'll change to lstrip

Reply:
fixed